cmd/kpromo/cmd/pr/pr.go

```diff
@@ -203,7 +203,7 @@ func runPromote(opts *promoteOptions) error {
 
 	// Path to the promoter image list
 	imagesListPath := filepath.Join(
-		image.ProdRegistry,
+		image.ProdRegistryLegacy,
```

Comment:
should we start using the new one and stop using the old?

Reply:
No, this requires changing the layout of the k/k8s.io repo and that's something that's way beyond scope of this PR.